src/dpl/src/Opendp.cpp

```diff
@@ -381,8 +381,9 @@ dbInst* Opendp::getAdjacentInstance(dbInst* inst, bool left) const
   dbInst* adjacent_inst = nullptr;
 
   // do not return macros, endcaps and tapcells
-  if (pixel != nullptr && pixel->cell && pixel->cell->db_inst_->isCore()) {
-    adjacent_inst = pixel->cell->db_inst_;
+  if (pixel != nullptr && pixel->cell
+      && static_cast<Cell*>(pixel->cell)->db_inst_->isCore()) {
+    adjacent_inst = static_cast<Cell*>(pixel->cell)->db_inst_;
```

Comment:
Why not use getDbInst rather than static_cast?  Similar in a few other places

Reply:
The same occurs on the line above
![image](https://github.com/user-attachments/assets/9fe40e73-d89f-49a2-9dfc-82eb06a491be)